adminer/content.md

```diff
@@ -30,6 +30,10 @@ Then point your web server to port 9000 of the container.
 
 Note: This exposes the FastCGI socket to the Internet. Make sure to add proper firewall rules or use a private Docker network instead to prevent a direct access.
 
+## %%STACK%%
+
+Run `docker stack deploy -c stack.yml %%REPO%%` (or `docker-compose -f stack.yml up`), wait for it to initialize completely, and visit `http://swarm-ip:8080`, `http://localhost:8080`, or `http://host-ip:8080` (as appropriate).
```

Comment:
@marcosnils Is this paragraph required to the online demo?

In any case, I don't *object* this change, but this paragraph refers to a downloaded swarm.yml which IMO is unnecessary clutter for adminer.